docs/expr-lang.ebnf

```diff
@@ -140,3 +141,12 @@ numeric_literal = ? clang::tok::numeric_constant ? ;
 boolean_literal = "true" | "false" ;
 
 pointer_literal = "nullptr" ;
+
+builtin_func = builtin_func_name "(" {builtin_func_argument_list} ")" ;
+
+builtin_func_name = "__log2" ;
+
+builtin_func_argument_list = builtin_func_argument
+                           | builtin_func_argument_list "," builtin_func_argument_list
```

Comment:
Is this supposed to be `builtin_func_argument "," builtin_func_argument_list`?

Reply:
Technically the other way around (`builtin_func_argument_list "," builtin_func_argument`), but yes, thanks for catching this!